mcs/build/library.make

```diff
@@ -26,6 +26,10 @@ LIB_REFS_ALIAS = $(filter-out $(LIB_REFS_FULL),$(LIB_REFS))
 LIB_MCS_FLAGS += $(patsubst %,-r:$(topdir)/class/lib/$(PROFILE_DIRECTORY)/%.dll,$(LIB_REFS_FULL))
 LIB_MCS_FLAGS += $(patsubst %,-r:%.dll, $(subst =,=$(topdir)/class/lib/$(PROFILE_DIRECTORY)/,$(LIB_REFS_ALIAS)))
 
+ifdef KEYFILE
+KEYFILE_MCS_FLAGS += /keyfile:$(KEYFILE)
```

Comment:
also there are a few stray cases of KEY_FILE e.g. in Facade, could you please consolidate them to KEYFILE?